modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/ExtensionComponent.java

```diff
@@ -154,6 +155,7 @@ protected ExtensionComponent(ExtensionModel extensionModel,
   @Override
   public final void initialise() throws InitialisationException {
     if (cursorProviderFactory == null) {
+      initialiseIfNeeded(streamingManager, muleContext);
```

Comment:
in which case isn't the streamingManager initialised? The runtime controls the lifecycle of this guy. This doesn't seem right